src/styles/main.scss

```diff
@@ -1,3 +1,311 @@
 body {
   margin: 0;
+  --card-margin-left-first: 41px;
+  --card-margin-left: 50px;
+  --card-max-width: 200px;
+  --card-border: 1px;
+  --card-border-radius: 5px;
+  --min-section-gap: 16px;
+  --min-section-top-gap: 16px;
+  --card-header-top-gap: 32px;
+  --card-margin: 0;
+  --card-padding: 0;
+  --card-padding-top: 40px;
+  --card-image-height: 134px;
+  --card-font-size: 12px;
+  --cardfont-color: #060b35;
+}
+
+.catalog {
+  margin: auto 0;
+  padding: 0 50px;
+  background-color: #fff;
+  box-shadow: 0 2 4 0 #000 5%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.logo__image {
+  height: 40px;
+  width: 40px;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  align-items: center;
+}
+
+.nav__item {
+  display: flex;
+  align-items: center;
+}
+
+.nav__link {
+  padding: 23px 0;
+  font-weight: 500;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  text-transform: uppercase;
+  color: #060b35;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+  text-transform: uppercase;
+}
+
+.nav__list :not(:last-child) {
+  margin-right: 20px;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  height: 4px;
+  width: 37px;
+  bottom: 0;
+  left: 0;
+  border-radius: 8px;
+  background-color: #00acdc;
+}
+
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  float: center;
+  margin-left: calc(
+    (100% -
+    (var(--card-max-width) * 4 + var(--card-margin-left) * 3))
+    / 2);
+  margin-right: calc(
+    (100% -
+    (var(--card-max-width) * 4 + var(--card-margin-left) * 3))
+    / 2);
+  max-width: calc(
+    var(--card-max-width) * 4
+    + var(--card-margin-left) * 3);
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+}
+
+.card {
+  box-sizing: border-box;
+  margin-top: 51px;
+  margin-bottom: 48px;
+  margin-left: var(--card-margin-left);
+  max-height: 408px;
+  width: var(--card-max-width);
+  background-color: #fff;
+  border: var(--card-border) solid #f3f3f3;
+  border-radius: var(--card-border-radius);
+}
+
+.main :first-child {
+  margin-left: 0;
+}
+
+.main :nth-child(5) {
+  margin-left: 0;
+  margin-top: 0;
+}
+
+.main :nth-child(6) {
+  margin-top: 0;
+}
+
+.main :nth-child(7) {
+  margin-top: 0;
+}
+
+.main :nth-child(8) {
+  margin-top: 0;
+}
```

Comment:
Don't repeat yourself. Use classes or `n` in brackets
https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child#examples